src/packages/galaxy/galaxy.py

```diff
@@ -60,6 +61,15 @@ class Galaxy:
     }
     MAX_PLOT_DISTANCE = 20000
 
+    MAX_RETRIES = 3
+    "The maximum number of times to retry a failed HTTP request before failing permanently."
+
+    TIMEOUT = ClientTimeout(total=10)
```

Comment:
[style] _blinks_ yeah id prefer `aiohttp.ClientTimeout`

I don't know what it is about these names, but i am getting very easily confused between these and our clients...

Reply:
704dd81